Dockerfile

```diff
@@ -1,60 +1,35 @@
-# Two stage Dockerfile to build OpenWPM
-# Stage 1 builds the extension, stage 2 builds the main OpenWPM image
-
-FROM node:10 as extension
-WORKDIR /usr/src/app
-
-# The extension needs to run for example the TypeScript transpiler
-# to generate the JavaScript code of the extension. This must be done as root
-# as long as the directory of the extension is only writeable as root.
-RUN npm config set unsafe-perm true
-
-COPY automation/Extension/firefox/. ./
-COPY automation/Extension/webext-instrumentation/. ../webext-instrumentation
-RUN npm install
-RUN npm run build
-RUN cp dist/openwpm-1.0.zip openwpm.xpi
-
-# Stage 2, build the main OpenWPM image
 FROM krallin/ubuntu-tini:bionic
 
-WORKDIR /opt/OpenWPM
+SHELL ["/bin/bash", "-c"]
```

Comment:
We need to verify that our signal propagation still works as expected when we do this, since bash doesn't propagate a SIGTERM properly but instead just SIGKILLS all child processes.
This might be a non issue, just raising awareness

Reply:
how would we verify this?

Reply:
I'd build the docker image and start docker compose
When the crawler gets scaled down the containers are send a SIGTERM
If the crawler receives a SIGTERM ([which the crawler logs](https://github.com/mozilla/OpenWPM/blob/2178e4c0ea641b95e861e336cf320a95b81402b3/crawler.py#L137) ) then we'd know the signal gets propagated as expected.

Reply:
I just realized we can't test just on your branch as issue-443 isn't merged yet.
I'll do that locally and report back

Reply:
I just checked and it gets propagated as expected